scipy/signal/_spectral_py.py

```diff
@@ -13,41 +13,50 @@
            'spectrogram', 'stft', 'istft', 'check_COLA', 'check_NOLA']
 
 
-def lombscargle(x,
-                y,
-                freqs,
-                precenter=False,
-                normalize=False):
+def lombscargle(
+    t: npt.NDArray,
+    y: npt.NDArray,
+    freqs: npt.NDArray,
+    precenter: bool = False,
+    normalize: bool | str = False,
```

Comment:
```suggestion
    normalize: bool | str = False,
    *,
```

Reply:
I'll add this as well.

Reply:
Added.

Reply:
Why not use
```python
normalize: bool | Literal['amplitude', 'power', 'normalize'] = False,
```
It let's mypy catch the typos.